test_radix.py

```diff
@@ -0,0 +1,58 @@
+from radix_sort import radixsort
+from random import shuffle
+import pytest
+
+
+def test_randoms():
+    x = range(100)
+    randoms = x[:]
+    shuffle(randoms)
+    radixsort(randoms)
+    y = randoms
```

Comment:
This is a wasted assignment.  You don't really need to have a second name for this version of the list.